sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestRecord.java

```diff
@@ -213,13 +214,26 @@ public long transportRequestId() {
     // region Methods
 
     public boolean expire() {
-        final RequestTimeoutException error = new RequestTimeoutException(this.toString(), this.args.physicalAddress());
+        final GoneException error = new GoneException(this.toString(), null, this.args.physicalAddress());
         BridgeInternal.setRequestHeaders(error, this.args.serviceRequest().getHeaders());
+
         return this.completeExceptionally(error);
     }
 
     public abstract Timeout newTimeout(final TimerTask task);
 
+    /**
+     * Provides information whether the request could have been sent to the service
+     * @return fals if it is possible to guarantee that the request never arrived at the service - true otherwise
```

Comment:
typo on return value :) 
`fals` -> `false`

Reply:
If I understand this correctly, we are saying that this API returns `true` if the request has been sent to the service and received it, `false` otherwise - meaning either the request is sent but not received by the service, or request is not sent at all ?

Reply:
No - will return true if there exists a chance that the request could have arrived at the service
False if we are 100% sure the request wasn't sent on the wire